common/protob/messages-cardano.proto

```diff
@@ -46,6 +46,11 @@ enum CardanoNativeScriptHashDisplayFormat {
     POLICY_ID = 2;
 }
 
+enum CardanoTxOutputSerializationFormat {
+    LEGACY = 0;
```

Comment:
Just noting this so we don't forget to rename it according to the discussion https://github.com/vacuumlabs/cardano-hw-interop-lib/pull/7#discussion_r895865454. There's also a mention of post-alonzo in the README so that should perhaps also be udpated when this gets renamed.

Reply:
Thanks for reminder, renamed in https://github.com/vacuumlabs/trezor-firmware/pull/78/commits/8c333516c61e2f0990dea73d0183f529b2d44998 and https://github.com/vacuumlabs/trezor-firmware/pull/78/commits/64eed36fc454b1098e16de8d055c92b387225abd. (I used only `legacy` and `babbage` in function and constant names, I find `array_legacy` and `map_babbage` too wordy for this purpose.)